src/test/java/gov/cancer/tests/crosscutting/Footer_Test.java

```diff
@@ -0,0 +1,544 @@
+package gov.cancer.tests.crosscutting;
+
+import java.util.Iterator;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import gov.cancer.framework.ExcelDataReader;
+import gov.cancer.pageobject.crosscutting.FooterPage;
+import gov.cancer.tests.TestObjectBase;
+import gov.cancer.tests.TestRunner;
+
+/**
+ * Tests for pages with footer.
+ */
+public class Footer_Test extends TestObjectBase {
+
+  /**
+   * This method is checking if the Footer exists on the pages
+   * 
+   * @param path
+   *          Path of the page to check.
+   */
+  @Test(dataProvider = "getPageFooterPaths")
+  public void footerIsVisible(String path) {
+
+    TestRunner.run(FooterPage.class, path, (FooterPage page) -> {
+
+      Assert.assertTrue(page.isFooterVisible(), "Page Footer is visible.");
```

Comment:
All of the Asserts should look like this. 👍 Just the page object, and a getter which returns a *simple* type.

Reply:
thanks